.github/workflows/main.yml

```diff
@@ -61,3 +61,45 @@ jobs:
         with:
           targets: ansible_collections/t_systems_mms/letsencrypt/roles/letsencrypt
           args: "-c ansible_collections/t_systems_mms/letsencrypt/.ansible-lint"
+
+  integration-tests:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ansible_collections/t_systems_mms/letsencrypt
+    strategy:
+      fail-fast: false
+      matrix:
+        ansible:
+          - stable-2.9
+          - stable-2.10
+        python:
+          - 2.7
+          - 3.7
+          - 3.8
+    steps:
+      - name: Check out code to collections-folder, so ansible finds it
+        uses: actions/checkout@v2
+        with:
+          path: ansible_collections/t_systems_mms/letsencrypt
+
+      - name: Run Pebble and challtestsrv
+        run: cd .. && curl https://raw.githubusercontent.com/letsencrypt/pebble/master/docker-compose.yml >docker-compose.yml && docker-compose up -d
```

Comment:
apparently you can pipe the docker-compose file https://github.com/docker/compose/issues/5120 :)

Reply:
... great.
I'll implement it. :)